Makefile

```diff
@@ -11,6 +11,12 @@ PKGS = $(shell go list ./... | grep -v /vendor/)
 gofmtcheck:
 	@go fmt $(PKGS) | grep ".*\.go"; if [ "$$?" = "0" ]; then exit 1; fi     
 
+doccheck:
+	grep -hoE '(kube_[^ |]+)' Documentation/* | sort -u > documented_metrics
```

Comment:
Nice suggestion @matthiasr! No I think that will make it feel more like a legitimate test case, so I think your suggestion makes a lot of sense and is not too verbose.